src/PKSim.Presentation/Services/EditMoleculeTask.cs

```diff
@@ -168,25 +149,26 @@ public ICommand EditMolecule(IndividualMolecule molecule, TSimulationSubject sim
          }
       }
 
-      private ICommand proteinFromQuery<TMolecule>(TSimulationSubject simulationSubject) where TMolecule : IndividualMolecule
-      {
-         using (_geneExpressionsDatabasePathManager.ConnectToDatabaseFor(simulationSubject.Species))
-         using (var presenter = _applicationController.Start<IProteinExpressionsPresenter>())
-         {
-            var moleculeFactory = _individualMoleculeFactoryResolver.FactoryFor<TMolecule>();
-            var newMolecule = moleculeFactory.AddMoleculeTo(simulationSubject, "%TEMP%");
-            presenter.InitializeSettings(_queryExpressionSettingsMapper.MapFrom(newMolecule, simulationSubject, ""));
-            presenter.Title = PKSimConstants.UI.AddProteinExpression(_executionContext.TypeFor(newMolecule));
-            if (!presenter.Start())
-            {
-               //needs to remove the molecule that was added previously
-               simulationSubject.RemoveMolecule(newMolecule);
-               return new PKSimEmptyCommand();
-            }
-
-            var queryResults = presenter.GetQueryResults();
-            return _moleculeExpressionTask.AddMoleculeTo(simulationSubject, newMolecule, queryResults);
-         }
-      }
+      //
+      // private ICommand proteinFromQuery<TMolecule>(TSimulationSubject simulationSubject) where TMolecule : IndividualMolecule
```

Comment:
I kept it for now but we don't have this use case anymore. We always EDIT a profile